src/python/python-proto/python_proto/tests/test_common.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+import uuid
+
+import pytest
+
+pytest.register_assert_rewrite("python_proto.tests.helpers")
```

Comment:
slick i had no idea this exists

Reply:
yeah it makes the failures nice to look at